Improve module install tests

mmathesius opened the pull request:
As noted in `module-install-1.feature`, the "I can install a module default profile with dependencies for an enabled module stream" scenario fails since dnf is apparently not tracking the version of ModuleD--even though a required package has been installed from it.

Martin suggest that required modules should only be enabled, but I think this is really a bug.

mmathesius committed:
Improve module install tests

r4f4 commented on dnf-docker-test/features/module-install-1.feature:
```diff
@@ -5,15 +5,123 @@ Feature: Installing module profiles
+
+  # currently know to fail since dnf does not track the version of ModuleD
+  # even though it has installed a package from it
+  Scenario: I can install a module default profile with dependencies for an enabled module stream
```
You can tag the scenario with `@xfail` in this case

kkaarreell replied:
I would rather just remove the failing version check, see my comment below.

kkaarreell commented on dnf-docker-test/features/module-install-1.feature:
```diff
@@ -5,15 +5,123 @@ Feature: Installing module profiles
+         | Key      | Value |
+         | enabled  | 1     |
+         | stream   | f26   |
+         | version  | 1     |
```
Please remove this particular line. As clarified by Petr, the module is just supposed to be enabled and therefore the version check doesn't make sense for a not-installed module.

mmathesius replied:
So, even though a required RPM has been installed from the required module, the module is still just considered "enabled" and not "installed"?

kkaarreell replied:
Exactly.

mmathesius committed:
Skip module version check since module is only enabled, not installed

mmathesius committed:
Remove module install tests that have been moved to different files

mmathesius commented:
Since this PR hasn't been merged yet, I just removed two tests that have been moved to new files (and rewritten) by PR https://github.com/kkaarreell/ci-dnf-stack/pull/7

kkaarreell commented:
Thank you Merlin.
